types/mui-datatables/index.d.ts

```diff
@@ -179,15 +186,15 @@ export interface MUIDataTableOptions {
             lookup: { [key: number]: boolean };
         },
         displayData: Array<{ data: any[]; dataIndex: number }>,
-        setSelectedRows: (rows: number[]) => void
+        setSelectedRows: (rows: number[]) => void,
     ) => React.ReactNode;
     disableToolbarSelect?: boolean;
     download?: boolean;
-    downloadOptions?: {
+    downloadOptions?: Partial<{
         filename: string;
         separator: string;
-        filterOptions?: { useDisplayedColumnsOnly: boolean; useDisplayedRowsOnly: boolean };
-    };
+        filterOptions: { useDisplayedColumnsOnly?: boolean; useDisplayedRowsOnly?: boolean };
```

Comment:
```suggestion
        filterOptions: Partial<{ useDisplayedColumnsOnly: boolean; useDisplayedRowsOnly: boolean }>;
```

If we're going to use the Partial helper type anyway